tests/unit/test_events.py

```diff
@@ -103,6 +106,11 @@ def github_issue_comment_propose_update(self):
         ) as outfile:
             return json.load(outfile)
 
+    @pytest.fixture()
+    def gitlab_issue_comment(self):
+        with open(DATA_DIR / "webhooks" / "gitlab" / "issue_comment.json") as outfile:
+            return json.load(outfile)
```

Comment:
Maybe, you can move this fixture near the other gitlab related ones to have them together.